subsys/dfu/src/dfu_target_modem.c

```diff
@@ -194,7 +259,7 @@ int dfu_target_modem_write(const void *const buf, size_t len)
 	int sent = 0;
 	int modem_error = 0;
```

Comment:
Why are these assigned zero and then overwritten unconditionally?

Reply:
I assume they shouldn't even have been initialized.